src/components/hw-design/auth/githubauth-button.tsx

```diff
@@ -0,0 +1,25 @@
+import { signIn } from "next-auth/react";
+import Image from "next/image";
+
+function GithubAuthButton({ redirect }: { redirect: string }) {
+  return (
+    <button
+      onClick={() => {
+        void signIn("github", { callbackUrl: redirect });
```

Comment:
why is void needed here btw? is it an eslint rule?

Reply:
yeah i think it was some sort of error iirc

Reply:
will double check tho

Reply:
right its cause im not handling the promise returned by signIn so it was being angry at me